src/wx/wxvbam.cpp

```diff
@@ -761,10 +766,20 @@ void MainFrame::OnMove(wxMoveEvent& event)
     (void)event; // unused params
     wxPoint pos = GetScreenPosition();
     int x = pos.x, y = pos.y;
-    if (x >= 0 && y >= 0 && !IsFullScreen())
+    if (!IsFullScreen() && !IsMaximized())
```

Comment:
This is the same check twice on Mac, but I don't think we need to have a macro check here to avoid one.